Tasks.php

```diff
@@ -344,7 +344,17 @@ private static function getNohupCommandIfPresent()
         if (SettingsServer::isWindows()) {
             return '';
         }
+        
+$requiredExecutablesCheck = StaticContainer::get(RequiredExecutablesCheck::class);
 
-        return 'nohup';
+        if ($requiredExecutablesCheck->isNohupPresent()) {
```

Comment:
@riczorn looks like you messed up the lines while applying the suggestion please fix it